docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -23,7 +32,211 @@ For simple spikes or tests, you can skip steps 1-2 and target any Bitbucket test
+
+#### Using the AWS Quick Start for Bitbucket
+
+If you are a new user, perform an end-to-end deployment. This involves deploying Bitbucket into a _new_ ASI:
+
+Navigate to **[AWS Quick Start for Bitbucket Data Center](https://aws.amazon.com/quickstart/architecture/bitbucket/) &gt; How to deploy** tab **&gt; Deploy into a new ASI** link.
+
+If you have already deployed the ASI separately by using the [ASI Quick Start](https://aws.amazon.com/quickstart/architecture/atlassian-standard-infrastructure/)ASI Quick Start or by deploying another Atlassian product (Jira, Bitbucket, or Confluence Data Center development environment) with ASI, deploy Bitbucket into your existing ASI:
+
+Navigate to **[AWS Quick Start for Bitbucket Data Center](https://aws.amazon.com/quickstart/architecture/bitbucket/) &gt; How to deploy** tab **&gt; Deploy into your existing ASI** link.
+
+{{% note %}}
+You are responsible for the cost of AWS services used while running this Quick Start reference deployment. This Quick Start doesn't have any additional prices. See [Amazon EC2 pricing](https://aws.amazon.com/ec2/pricing/) for more detail.
+{{% /note %}}
+
+To reduce costs, we recommend you to keep your deployment up and running only during the performance runs.
+
+#### AWS cost estimation for the development environment
+
+AWS Bitbucket Data Center development environment infrastructure costs about 20 - 40$ per working week depending on such factors like region, instance type, deployment type of DB, and other.  
+
+#### <a id="quick-start-parameters"></a> Quick Start parameters for development environment
+
+All important parameters are listed and described in this section. For all other remaining parameters, we recommend using the Quick Start defaults.
+
+**Bitbucket setup**
+
+| Parameter | Recommended value |
+| --------- | ----------------- |
+| Bitbucket Product | Software |
+| Version | The Data Center App Performance Toolkit officially supports `7.6.7`, `6.10.11` ([Long Term Support releases](https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html)) and `7.0.5` Platform Release |
+
+**Cluster nodes**
+
+| Parameter | Recommended value |
+| --------- | ----------------- |
+| Cluster node instance type | [t3.medium](https://aws.amazon.com/ec2/instance-types/t3/) (we recommend this instance type for its good balance between price and performance in testing environments) |
+| Maximum number of cluster nodes | 1 |
+| Minimum number of cluster nodes | 1 |
+| Cluster node instance volume size | 50 |
+
+**File server**
+
+| Parameter | Recommended Value |
+| --------- | ----------------- |
+| File server instance type | m4.xlarge |
+| Home directory size | 1000 |
```

Comment:
Let's use minimum possible value for this field: `100`

Reply:
Okay, lets try